app/Models/IntegrationRelease.php

```diff
@@ -16,19 +18,52 @@ class IntegrationRelease extends BaseModel implements HasMedia
     use InteractsWithMedia;
     use HasStabilityFlags;
 
+    use LogsActivity {
+        LogsActivity::activities as auditLog;
+    }
+
     protected $table = 'integration_releases';
 
     protected $fillable = [
         'minimum',
         'notes',
         'stable',
         'version',
+        'created_at',
     ];
 
     protected $with = [
         'media',
     ];
 
+    protected static function booted()
+    {
+        static::saving(function (IntegrationRelease $release) {
+            if ($release->minimum) {
+                // Set all other integration releases to not be the minimum.
+                // There can only be one minimum.
+                static::where('id', '!=', $release->id)
+                    ->update(['minimum' => false]);
+            }
+        });
```

Comment:
What happens if my order of operations is:
1. Manually unset the current minimum.
2. Spend a minute or two navigating to the new minimum's form.
3. Turn on the new minimum.

During that minute or two, would clients see any disruption?

Reply:
The client call isn't hooked up yet. I was looking at including that in the LatestClient PR, but it's still reading from the file.  I thought it would be better to convert it to use the database, so I threw this together.

If no minimum is sent to the client, the minimum version will effectively be "0.0.0.0", and all versions will be allowed during that window.

Having said that, the more natural flow is just to assign a new minimum, which will unset the old minimum. That's the same behavior we already have for emulators.